server/src/tests/cmdline_utils_spec.ts

```diff
@@ -33,4 +33,24 @@ describe('parseCommandLine', () => {
     const options = parseCommandLine(['--tsProbeLocations', '/baz,/qux']);
     expect(options.tsProbeLocations).toEqual(['/baz', '/qux']);
   });
+
+  it('should parse without "includeCompletionsForModuleExports"', () => {
+    const options = parseCommandLine(['--tsProbeLocations', '/baz,/qux']);
+    expect(options.includeCompletionsForModuleExports).toEqual(true);
+  });
```

Comment:
Add a test case when the config `includeCompletionsForModuleExports` doesn't exist.